assets/src/components/mapPage.tsx

```diff
@@ -69,18 +70,16 @@ const OldSearchResultList = ({
 }
 
 const SearchMode = ({
-  selectSearchResult,
+  selectVehicleResult,
+  selectLocationResult,
 }: {
-  selectSearchResult: (result: Vehicle | Ghost | null) => void
+  selectVehicleResult: (result: Vehicle | Ghost | null) => void
+  selectLocationResult: (result: LocationSearchResult | null) => void
```

Comment:
suggestion(non-blocking): perhaps prefix these with `on`?

Since working on the autocomplete, I've been trying to align event names with those in html (i.e., "on" + present tense description, like `onSubmit`)

Reply:
looks great, thank you!